rust/may_minihttp/Cargo.toml

```diff
@@ -4,4 +4,13 @@ version = "0.0.0"
 edition = "2021"
 
 [dependencies]
-may_minihttp = "0.1"
\ No newline at end of file
+may_minihttp = { version = "0.1", default-features = false }
+
+[profile.release]
+opt-level = 3
```

Comment:
can we use this as command line arguments ?

Reply:
other rust frame work also use this value